frontend/src/worksheets/WorksheetForm.js

```diff
@@ -91,63 +99,73 @@ export default function WorkSheetForm({
                   )}
                   <SelectInput
+                    disabled={disabled}
                     name="assetSettlement"
                     label="Eszközök elszámolás módja"
                     container={ASSET_SETTLEMENT_LIST}
                   />
                   <SelectInput
+                    disabled={disabled}
                     name="workingTimeAccounting"
                     label="Munkaidő elszámolás módja"
                     container={WORKING_TIME_ACCOUNT_LIST}
                   />
                   <Input
+                    disabled={disabled}
                     name="numberOfEmployees"
                     label="Létszám"
                     type="number"
                     min="0"
                   />
                   <Input
+                    disabled={disabled}
                     name="overheadHour"
                     label="Rezsióra"
                     type="number"
                     min="0"
                   />
                   <Input
+                    disabled={disabled}
                     name="deliveryKm"
                     label="Kiszállás"
                     type="number"
                     min="0"
                   />
-                  <Input
-                    name="accountSerialNumber"
-                    label="A munkalaphoz tartozó számla sorszáma"
-                    type="text"
-                  />
                   <TextareaInput
+                    disabled={disabled}
                     name="description"
                     label="Elvégzett munka leírása"
                   />
                   <Input
+                    disabled={disabled}
                     name="usedMaterial"
                     label="Felhasznált anyagok"
                     type="text"
                   />
                   <SelectInput
+                    disabled={disabled}
                     name="typeOfPayment"
                     label="Fizetés módja"
                     container={TYPE_OF_PAYMENT_LIST}
                   />
+                  <Input
+                    disabled={isReported ? false : disabled}
+                    name="accountSerialNumber"
+                    label="A munkalaphoz tartozó számla sorszáma"
+                    type="text"
+                  />
                   <span>Kelt: </span>
                   <CalendarDropDown
+                    disabled={disabled}
                     name="createdAt"
                     setFieldValue={setFieldValue}
                     value={values.createdAt}
                   />
-                  <div className="mt-3">
+                  <div className={disabled ? 'hidden' : 'mt-3'}>
                     Munkát elvégezte:
                     <Signature name="workerSignature" />
                   </div>
-                  <div className="mt-3">
+                  <div className={disabled ? 'hidden' : 'mt-3'}>
```

Comment:
Ezt szerintem jó lenne továbbra is megjeleníteni, csak letiltani a szerkesztést.

Reply:
PM-mel egyeztessetek erről.

Reply:
Szerintem ez így van. A hidden class nem beépített, így néz ki a style.css-ben:
.hidden {
  opacity: 0.2;
  pointer-events: none;
}

Reply:
Az elrejtést értem, de a pointer-events: none miatt nem is lehet akkor átrajzolni?
B kérdés: Törlés gombbal nem is lehet kitörölni az előző aláírást?

Reply:
Ja, bocsi, nem láttam, hogy ez az aláírásnál van. Ebben az esetben majd Dani válaszol a fenti kérdésekre.

Reply:
Én úgy értelmeztem, ha készre van jelentve (azaz aláírta mindkét résztvevő) akkor már nem kell az aláírást módosítani. De akkor utána kérdezek.

Reply:
csak létrehozott állapotban lehet aláírni, meg értelem szerűen új felvételénél.